network/evpn-gw/evpn-gw-cni/NOTICE

```diff
@@ -0,0 +1,12 @@
+EVPN GW CNI
+Copyright (C) 2023 Nordix Foundation
```

Comment:
The naming and design for grpc split - xPU inframanager  came from IPDK.io. Not sure we can slap a copyright like this here.

Reply:
For which specific code are you refering to that I need to pass copyright. I do not understand exactly from your comment above